### PR TITLE
Adding Simkl integration

### PR DESCRIPTION
**Pull request by Freaksed**
#### Database Migration
NO

#### Description
Beginning integration for simkl.com
First steps have been to duplicate trakt integrations and update references and api keys

#### Issues Fixed or Closed by this PR

* #

### CODE REVIEW

**Comment by marine1988** [comment]
please accept

**Comment by austinwbest** [comment]
> please accept

There is nothing to accept bud. It was noted as "started" 2 months ago but nothing since.

**Comment by AngelyChris** [comment]
will this be implemented to v3? if so, im looking forward to this addition. cheers

File: src/NzbDrone.Core/Notifications/Simkl/SimklService.cs [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+        private string MapAudioChannels(MovieFile movieFile, string audioFormat)
+        {
+            var audioChannels = movieFile.MediaInfo != null ? MediaInfoFormatter.FormatAudioChannels(movieFile.MediaInfo).ToString("0.0") : string.Empty;
+
+            // Map cases where Radarr doesn't handle MI correctly, can purge once mediainfo handling is improved
+            if (audioChannels == "8.0")
+            {
+                audioChannels = "7.1";
+            }
+            else if (audioChannels == "6.0" && audioFormat == "dts_ma")
```
Comment (Qstick) [inline src/NzbDrone.Core/Notifications/Simkl/SimklService.cs:251]: Alot of this was fixed in the mediainfo implementation, you may want to sync back up with the trakt file

File: src/NzbDrone.Core/Notifications/Simkl/SimklProxy.cs [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        void RemoveFromCollection(SimklCollectMoviesResource payload, string accessToken);
+        HttpRequest BuildSimklRequest(string resource, HttpMethod method, string accessToken);
+    }
+
+    public class SimklProxy : ISimklProxy
+    {
+        private const string URL = "https://simkl.com";
+        private const string OAuthUrl = "https://simkl.com/oauth/authorize";
+        private const string RedirectUri = "https://auth.servarr.com/v1/simkl/auth";
+        private const string RenewUri = "https://auth.servarr.com/v1/simkl/renew";
```
Comment (Qstick) [inline src/NzbDrone.Core/Notifications/Simkl/SimklProxy.cs:24]: @ta264 will need to add support for this.

**Review by Qstick — CHANGES_REQUESTED** [review]
This appears to be an exact copy of the trakt code, are we saying nothing is different? Also, I have no way to test this fully until we get auth hooked up. 

One note on the audio channels that needs fixed

**Comment by bakerboy448** [comment]
Closing; appears OP has abandoned; can reopen if needed

**Pull request closed without merging**